e2e/README.md

```diff
@@ -0,0 +1,41 @@
+# E2E SDK Automation
+
+### Node version
+`nvm use 18`
+
+### Setup:
+
+`yarn setup`
+
+### Set a test SRP:
+`export SRP=TEST TEST TEST TEST TEST TEST TEST TEST TEST TEST TEST TEST`
+
+### Env files needed:
+
+.ios.env:
```

Comment:
create a `.ios.env.example`, `.android.env.example`, `.dapps.env.example` and `.env.example` with those field as a referral

Reply:
Done